tests/Feature/Cache/SettingCacheTest.php

```diff
@@ -210,7 +215,7 @@ public function testTryClearByPatternWithNonRedisDriver()
     {
         config()->set('cache.default', 'array');
 
-        $this->expectException(SettingCacheException::class);
+        $this->expectException(\Exception::class);
```

Comment:
Why this was changed to a generic Exception, seems it should return SettingCacheException

Reply:
My mistake, it was fixed